.babelrc

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
- [ ] It would be better if we align or position the user icon to the booklist component. Here is a screenshot showing what we did and what is expected of us
Project
![Screenshot from 2021-09-07 08-34-04](https://user-images.githubusercontent.com/46388113/132313012-b7ed43e2-e9de-47ab-95d5-e726a86b5157.png)
Required
![Screenshot from 2021-09-07 08-30-45](https://user-images.githubusercontent.com/46388113/132312984-4cbd17aa-df37-4296-8ae9-f8c7cf7f4dce.png)